src/views/home/index.tsx

```diff
@@ -204,7 +211,9 @@ export default function HomeViewAsyncWrapper() {
       error={(error) => (
         <Fill>
           <Alert color="red">
-            An error occured while loading this demo directory: {String(error)}
+            {/* NOTE: this shouldn't happen unless no home directory was found */}
+            An error occurred while scanning for demo files. Is the demo storage
+            directory set?
```

Comment:
I would leave the explicit error message, since errors can still happen even if the storage directory is set.
For example, the user could be missing the permission to read the directory, the directory does not exist, and so on.